HomeTask5/src/Controller.js

```diff
@@ -0,0 +1,92 @@
+export default class Controller {
+    constructor(model, view) {
+        this._model = model;
+        this._view = view;
+    }
+
+    selectCategory = (category) =>{
+            for (var i = 1; i < 11; i++) {
+                if ((category === 0 && i === 1) || category === i) {
+                    let tittle = document.getElementById("tittle" + i);
+                    tittle.style.display = "block";
+                    tittle.className = 'section';
+                }
+                else if(category === 0) {
+                    let tittle = document.getElementById("tittle" + i);
+                    tittle.style.display = "block";
+                    tittle.className = 'section line';
+                }
+                else {
+                    let tittle = document.getElementById("tittle" + i);
+                    tittle.style.display = "none";
+                }
+            }
+    }
+
+    openPopUp = (value) =>{
+        // getter example
+        var news = {
+            text: "Nor is there anyone who loves or pursues or desires to obtain pain of itself, because it is pain, but occasionally circumstances occur in which toil and pain can procure him some great pleasure.",
+            get getText() {
+                return this.text;
+            }
+        };
+
+        var modal = document.getElementById("myModal");
+        modal.style.display = "block";
+        let text = news.getText;
+        document.getElementById("dynamictittle").innerHTML = document.getElementById("ptittle" + value).innerHTML;
```

Comment:
document query selector should not be a part of angular code. try to get rid of it

Reply:
This is not angular task. This is web pack& babel task code.